src/Workspaces/Core/Portable/Workspace/Solution/ProjectChanges.cs

```diff
@@ -117,15 +117,36 @@ public IEnumerable<DocumentId> GetAddedAdditionalDocuments()
         }
 
         public IEnumerable<DocumentId> GetChangedDocuments()
+        {
+            return GetChangedDocuments(false);
+        }
+
+        /// <summary>
+        /// https://github.com/dotnet/roslyn/issues/18199
```

Comment:
Given this is what will appear to people using our API, we _probably_ don't want to give them a GitHub bug. Care to write a better description?

Reply:
updated